testing/cli-e2etest/testscenarios/transaction/list_transactions_test.go

```diff
@@ -134,10 +134,11 @@ func TestListTransactions(t *testing.T) {
 		result := tracetestcli.Exec(t, "list transaction --sortBy name --sortDirection asc --output json", tracetestcli.WithCLIConfig(cliConfig))
 		helpers.RequireExitCodeEqual(t, result, 0)
 
-		transactions := helpers.UnmarshalJSON[[]types.AugmentedTransactionResource](t, result.StdOut)
-		require.Len(transactions, 3)
+		transactions := helpers.UnmarshalJSON[types.ResourceList[types.AugmentedTransactionResource]](t, result.StdOut)
```

Comment:
this scenario was incorrect (as well as the old implementation). it was returning an array of resources instead of a `ResourceList`